repose-aggregator/core/core-lib/pom.xml

```diff
@@ -137,6 +138,18 @@
 
         <dependency>
             <groupId>org.openrepose</groupId>
+            <artifactId>atom-feed-service-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
```

Comment:
Well, the same thing was done for the datastore service.

My guess is that including both as dependencies gets the classes/interfaces from both into the core-lib JAR. core-lib itself probably doesn't need access to the service.